examples/v1beta1/github-add-changed-files-pr/README.md

```diff
@@ -0,0 +1,47 @@
+## GitHub EventListener
+
+Creates an EventListener that listens for GitHub webhook events.
```

Comment:
Could you change this line to give more info about add-changed file info

Reply:
Updated